components/forms/CreateCompanyForm.tsx

```diff
@@ -0,0 +1,50 @@
+import { Form, Input, Modal } from 'antd';
+import companiesApi from '../../api/companiesApi';
+import { CompanyData } from '../../types/company';
+
+type Props = {
+  isOpen: boolean;
+  closeForm: () => void;
+  onCreate: (company: CompanyData) => void;
+};
+
+function CreateCompanyForm({ isOpen, closeForm, onCreate }: Props) {
+  const [form] = Form.useForm();
+
+  const onSubmit = () => {
+    form.validateFields().then((values) => {
+      companiesApi.createCompany(values).then((resp) => {
+        if (resp.payload.id === undefined) {
+          return;
+        }
+
+        form.resetFields();
+        closeForm();
+        onCreate(resp.payload as CompanyData);
+      });
+    });
+  };
+
+  return (
+    <Modal title={'Add new company'} open={isOpen} onOk={onSubmit} onCancel={closeForm}>
```

Comment:
```suggestion
    <Modal title={'Add a new organization'} open={isOpen} onOk={onSubmit} onCancel={closeForm}>
```

Reply:
Rip I just realised it might be better to put organization instead of company (e.g. govtech / mindef etc), wdyt?